### PR TITLE
Reintroduce bazel

### PR DESCRIPTION
**Pull request by justinsb**
It was removed along the CRDification

```release-note
NONE
```

### CODE REVIEW

File: pkg/apis/cluster/v1alpha1/defaults.go~ [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/*
```
Comment (roberthbailey) [inline pkg/apis/cluster/v1alpha1/defaults.go~]: This file shouldn't be in here.... Can you make sure there aren't any other temp files?

Comment (justinsb) [reply pkg/apis/cluster/v1alpha1/defaults.go~]: Gah - sorry, will fix

**Comment by justinsb** [comment]
Rebased, regenerated, removed temp files, repushed

**Pull request merged**